packages/frontpage/app/(app)/post/new/_action.ts

```diff
@@ -22,6 +22,13 @@ export async function newPostAction(_prevState: unknown, formData: FormData) {
     return { error: "Title too long" };
   }
 
+  const allowedProtocols = ["http", "https", "at"];
```

Comment:
We need to do more work to support `at://` URIs, that's captured in UN-81. For this PR let's just ignore that feature for now.

Reply:
To clarify, let's not accept at:// URIs. Just http(s)

Reply:
Yeah ok, I was trying to run before I could walk here.